bigtable/admin.go

```diff
@@ -247,12 +313,13 @@ type FamilyInfo struct {
 	GCPolicy string
 }
 
-// TableInfo retrieves information about a table.
-func (ac *AdminClient) TableInfo(ctx context.Context, table string) (*TableInfo, error) {
+func (ac *AdminClient) getTable(ctx context.Context, table string) (*btapb.Table, error) {
 	ctx = mergeOutgoingMetadata(ctx, ac.md)
 	prefix := ac.instancePrefix()
 	req := &btapb.GetTableRequest{
 		Name: prefix + "/tables/" + table,
+		View: btapb.Table_FULL,
+		// View: btapb.Table_ENCRYPTION_VIEW,
```

Comment:
What's the difference between these? Should you provide some kind of switch for callers?